src/react-app-env.d.ts

```diff
@@ -1 +1 @@
-/// <reference types="react-scripts" />
+/// <reference types='react-scripts' />
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.